### PR TITLE
Config netty param in HttpServer

### PR DESCRIPTION
**Pull request by liutang123**
for #4224
Add three http param to control the netty behavior.

### CODE REVIEW

**Review by kangkaisen — DISMISSED** [review]
+1

**Comment by kangkaisen** [comment]
@liutang123 Hi. please add these new configs to doc.

File: fe/fe-core/src/main/java/org/apache/doris/http/HttpServer.java [hunk 1]
```diff
@@ -17,6 +17,17 @@
 
 package org.apache.doris.http;
 
+import io.netty.bootstrap.ServerBootstrap;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/http/HttpServer.java:20]: remain the import order.

Comment (imay) [reply fe/fe-core/src/main/java/org/apache/doris/http/HttpServer.java:20]: We'd better to provide some config for popular IDE

**Comment by liutang123** [comment]
> @liutang123 Hi. please add these new configs to doc.

Because `maxChunkSize` is removed after netty 5.x, so I don't add it to docs and just add the other two.

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**